ivadomed/scripts/training_curve.py

```diff
@@ -9,26 +9,30 @@
 from textwrap import wrap
 from tensorboard.backend.event_processing.event_accumulator import EventAccumulator
 
-from ivadomed.utils import init_ivadomed
+from ivadomed.utils import init_ivadomed, Metavar
 
 
 def get_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("-i", "--input", required=True, type=str,
-                        help="Input log directory. If using --multiple, this parameter indicates the suffix path of all"
-                             " log directories of interest. To compare trainings or set of trainings (using "
-                             "``--multiple``) with subplots, please list the paths by separating them with commas, eg "
-                             "path_log_dir1,path_logdir2.")
+                        help="""Input log directory. If using --multiple, this parameter indicates
+                                the suffix path of all log directories of interest. To compare
+                                trainings or set of trainings (using ``--multiple``) with subplots,
+                                please list the paths by separating them with commas, eg
+                                path_log_dir1,path_logdir2.""",
+                        metavar=Metavar.str)
     parser.add_argument("--multiple", required=False, dest="multiple", action='store_true',
                         help="Multiple log directories are considered: all available folders with -i as "
                              "prefix. The plot represents the mean value (hard line) surrounded by the standard "
                              "deviation envelope.")
     parser.add_argument("-y", "--ylim_loss", required=False, type=str,
-                        help="Indicates the limits on the y-axis for the loss plots, otherwise these limits are "
-                             "automatically defined. Please separate the lower and the upper limit by a comma, eg -1,0."
-                             "Note: for the validation metrics: the y-limits are always 0.0 and 1.0.")
+                        help="""Indicates the limits on the y-axis for the loss plots, otherwise
+                                these limits are automatically defined. Please separate the lower
+                                and the upper limit by a comma, eg -1,0. Note: for the validation
+                                metrics: the y-limits are always 0.0 and 1.0.""",
+                        metavar=Metavar.str)
```

Comment:
```suggestion
                        metavar=Metavar.float)
```

Reply:
Right! thanks will fix that :)

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/641/commits/bc1691bdb29c57f758096efd0731c57e21580d82